cypress/support/commands.js

```diff
@@ -22,7 +22,7 @@ const login = () => {
 
   cy.get('.login-card input[name=password]')
     .clear()
-    .type(Cypress.env('INTEGRATION_PASSWORD'));
+    .type(Cypress.env('INTEGRATION_PASSWORD') || '1234');
```

Comment:
Need to find a way how to pass this Environment Variable to Cypress when running our tests in BS...